style.css

```diff
@@ -103,264 +83,187 @@
-    font-weight: 500;
-    font-size: 17px;
-    line-height: 24px;
+  .techLanguage ul li {
     display: flex;
-    align-items: center;
-    text-align: center;
-    letter-spacing: 0.03em;
-    color: #396df2;
-    border: 1px solid #396df2;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+  }
+
+  .techLanguage ul li img {
+    width: 30px;
+    height: 30px;
     position: relative;
-    left: 20px;
-    margin-top: 10px;
-    margin-bottom: 10px;
-    padding: 7px 20px;
-    border-radius: 8px;
+    top: 10px;
   }
 
-  .contactLink .resum:hover {
-    background: #396df2;
-    color: white;
+  .Contacting {
+    font-family: "Poppins", sans-serif;
+    background-color: #6070ff;
+    text-align: center;
+    height: 846px;
+    padding: 24px;
+    border-top-left-radius: 80px;
+    background-image: url("img/contact-mobile.svg");
+    background-position: right 50px;
+    background-repeat: no-repeat;
   }
 
-  .framWork ul li {
-    display: flex;
-    flex-direction: row;
-    color: brown;
-    gap: 15px;
-    margin-bottom: 10px;
-    margin-left: 20px;
-    width: 303px;
-    height: 66.72px;
-    padding: 8px 20px;
-    background: #f7f7f9;
-    border-radius: 8px;
+  .Contacting h1 {
+    color: #fff;
+    font-size: 40px;
+    font-weight: 700;
+    margin-top: 103px;
   }
 
-  .framWork p {
-    color: #172b4d;
-    position: relative;
-    top: 20px;
+  .Contacting p {
+    font-size: 20px;
+    font-weight: 400;
+    color: #ebebff;
   }
 
-  .framWork .code {
-    display: flex;
-    gap: 180px;
-    margin-top: 20px;
-    margin-bottom: 50px;
-    position: relative;
-    left: 40px;
+  .formController {
+    margin: 8px auto;
   }
 
-  .framWork .code img {
-    width: 5%;
-    height: 5%;
-    position: relative;
-    top: 10px;
+  .formController input {
+    width: 95%;
+    height: 48px;
+    border-radius: 8px;
+    border: none;
+    outline: none;
+    padding: 4px 8px;
   }
 
-  .techLanguage ul li {
-    display: flex;
-    gap: 180px;
-    margin-top: 20px;
-    position: relative;
-    left: 40px;
-    margin-bottom: 40px;
-    border-bottom: 2px solid #dfe1e6;
-    padding-bottom: 20px;
-    width: 300px;
+  .formController textarea {
+    width: 95%;
+    height: 170px;
+    border-radius: 8px;
+    border: none;
+    outline: none;
+    padding: 4px 8px;
   }
 
-  .techLanguage #oneImg {
-    width: 30px;
-    height: 30px;
+  .Contacting .btn {
+    display: block;
+    height: 48px;
+    width: 142px;
+    border-radius: 8px;
+    color: #6070ff;
+    font-size: 17px;
+    border: 1px solid #6070ff;
   }
 
-  .techLanguage .move {
-    position: relative;
-    left: 50px;
+  .Contacting .btn:hover {
+    background: #6070ff;
+    color: #fff;
   }
 
-  .techLanguage .oneMove {
-    position: relative;
-    left: -12px;
+  .Contacting .btn:enabled {
+    background: #fff;
+    color: #6070ff;
+  }
+
+  .Contacting .btn:disabled {
+    background: #fff;
+    color: #5e6c84;
```

Comment:
- [OPTIONAL] You have done a great job so far implementing the hover states suggested by the previous reviewers. :+1: :100: :+1:  You have added the correct code for the hover state but the code `.Contacting .btn:enabled` is preventing it from working. I'll suggest you change the `enabled` to `active` as that's the correct state when a button is clicked. Please make sure you change that online of code to the suggested one as demonstrated in the video below before merging this pull request. :smile: :small_airplane: 

- VIDEO REFERENCE AND DEMO:

https://user-images.githubusercontent.com/107632780/235744221-be1590d6-91e4-4f5d-9b67-d6cc75dd4abc.mp4